### ISSUE TITLE
POWER9 CPU is not recognised in /system/hwinfo page

### ISSUE THREAD

**Issue by runlevel5** [cockpit-project/cockpit#15563]
Cockpit version: 239-1.fc33
OS: Fedora 33 Server (running under QEMU + KVM)
Page:

```
cat /proc/cpuinfo
processor	: 0
cpu		: POWER9 (architected), altivec supported
clock		: 3000.000000MHz
revision	: 2.3 (pvr 004e 1203)

processor	: 1
cpu		: POWER9 (architected), altivec supported
clock		: 3000.000000MHz
revision	: 2.3 (pvr 004e 1203)

processor	: 2
cpu		: POWER9 (architected), altivec supported
clock		: 3000.000000MHz
revision	: 2.3 (pvr 004e 1203)

processor	: 3
cpu		: POWER9 (architected), altivec supported
clock		: 3000.000000MHz
revision	: 2.3 (pvr 004e 1203)

processor	: 4
cpu		: POWER9 (architected), altivec supported
clock		: 3000.000000MHz
revision	: 2.3 (pvr 004e 1203)

processor	: 5
cpu		: POWER9 (architected), altivec supported
clock		: 3000.000000MHz
revision	: 2.3 (pvr 004e 1203)

processor	: 6
cpu		: POWER9 (architected), altivec supported
clock		: 3000.000000MHz
revision	: 2.3 (pvr 004e 1203)

processor	: 7
cpu		: POWER9 (architected), altivec supported
clock		: 3000.000000MHz
revision	: 2.3 (pvr 004e 1203)

timebase	: 512000000
platform	: pSeries
model		: IBM pSeries (emulated by qemu)
machine		: CHRP IBM pSeries (emulated by qemu)
MMU		: Radix
```

However the `/system/hwinfo` page seems unable to pick up the POWER9 CPU info correctly:

![Screenshot from 2021-03-21 08-28-06](https://user-images.githubusercontent.com/135605/111886108-deb5ae80-8a1f-11eb-88f2-3679a09ff360.png)

**Comment by martinpitt** [comment]
Thanks for the report! I'll fix that.

**Comment by martinpitt** [comment]
Fixed in PR #15564
